server/index.ts

```diff
@@ -265,6 +312,9 @@ closeWithGrace(async () => {
 	await new Promise((resolve, reject) => {
 		server.close(e => (e ? reject(e) : resolve('ok')))
 	})
+	await new Promise((resolve, reject) => {
+		metricsServer.close(e => (e ? reject(e) : resolve('ok')))
```

Comment:
Does this cause a problem if we never call listen on this server? Maybe this should be in the if as well?

Reply:
It doesn't case a problem, but I can wrap it still.

Reply:
Actually I didn't like it so I took a bit different approach. I know we were on the goal line, but this feels cleaner than if's all over checking the .env